src/cyw43_ll.c

```diff
@@ -1405,7 +1405,7 @@ static void cyw43_clm_load(cyw43_int_t *self, const uint8_t *clm_ptr, size_t clm
         #pragma GCC diagnostic pop
         memcpy(buf + 20, clm_ptr + off, len);
 
-        CYW43_VDEBUG("clm data send %u/%u\n", (int)(off + len), (int)clm_len);
+        CYW43_VDEBUG("clm data send %u/%zu\n", off + len, clm_len);
```

Comment:
This was recently changed in a1c55f8767d3b45a414481dd8c49ced89df04766 to not use %z formatting, which isn't always implemented on embedded targets.  So please leave this as-is.